float64 k1
float64 k2
float64 k3
---
string result
